.github/workflows/rack-continuous.yaml

```diff
@@ -1,58 +1,58 @@
-# Lints the RACK source
+# Lints the RACK source upon every push (while ignoring tag-only pushes)
 
 name: RACK continous lint
-on: [push]
+
+on:
+  pull_request:
+    branches: [ '*' ]
+  push:
+    branches: [ '*' ]
+    tags-ignore: [ '*' ]
```

Comment:
Likewise, now we will lint files on both pull requests and pushes while ignoring tag-only pushes.